ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -58,7 +58,7 @@ conformance_test(
         "@curl_nix//:bin/curl",
         "@grpcurl_nix//:bin/grpcurl",
         "@jq_dev_env//:jq",
-        "@jdk11_nix//:bin/java",
+        "@bazel_tools//tools/jdk",
```

Comment:
Yes, it could perhaps even be simplified to importing canton as a `java_binary` since that would use the correct JDK version as well. Could avoid some of the dance around `$JAVA ...`. There are other instances of this pattern in the code base where that may also be possible.